tox.ini

```diff
@@ -34,6 +34,14 @@ deps=
 commands=
     python -m pytest tests/quality/
 
+
+[testenv:quality]
```

Comment:
Needs to be `[testenv:quality2]` - as-is, we're just seeing Tox errors instead of tests.